### DISCUSSION TITLE
Can I add my own aliases for my website?

### DISCUSSION THREAD

**Discussion by MuelNova** [facebook/docusaurus#9499] [Q&A] [answered]
instead of `@site/static/image/svg`, I want to add `@svg` (for example) points to my "website/static/image/svg" folder, is there any way that I can do it?

I have tried `webpack` in `docusaurus.config.js`, but it says `ValidationError: Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.`
```typescript
webpack: {
      jsLoader: (isServer) => {
        return {
          resolve: {
            alias: {
              myAlias: "myAlias"
            }
          }
        }
      }
    }
```

I also see [configureWebpack](https://docusaurus.io/docs/api/plugin-methods/lifecycle-apis#configureWebpack), but not sure if that works.

I've searched for a lot of keywords, but no results. I'm new to TypeScript and don't know what to do :(

**Comment by slorber** [comment, marked as answer]
This is not related to typescript.
You have to use configureWebpack through a plugin, and this config isn't supposed to be returned by jsLoader
